personio-personnel-data-api-oa3.yaml

```diff
@@ -1267,6 +1277,180 @@ paths:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates a project into the company account
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                  example: "A project name"
+      responses:
+        "200":
+          description: Project created successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    $ref: "#/components/schemas/Project"
+        "400":
+          description: Invalid data was sent
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: false
+                  error:
+                    type: object
+                    properties:
+                      name:
+                        type: array
+                        items:
+                          type: string
+                          example: "Please introduce a name to continue"
+
+  /company/attendances/projects/{id}:
+    delete:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates projects into the company account
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: Numeric `id` of the project
+          schema:
+            type: integer
+            format: int32
+      responses:
+        "204":
+          description: Project was successfully deleted
+        "404":
+          description: Project does not exist
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: false
+                  message:
+                    type: string
+                    example: "Project 5 not found"
+
+    patch:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Updates a project with the given data
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: Numeric `id` of the project
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                  example: "A project name"
+                active:
+                  type: boolean
+                  description: "Marks if the project is available for time tracking"
```

Comment:
As talked about, if tomorrow we add projects to absences, for example, that description is confusing or it can give the impression that projects in absences are different than projects in attendances (where they are the same) ... So instead of making the description explicit about time tracking you would write something like `Marks if the project is available for listing/usage`

Reply:
But that's not what's happening on Personio at the moment 😓. Also, the GET get's all the projects, regardless if they are active or not. Let's discuss it later on the daily